app/src/main/java/nodomain/freeyourgadget/gadgetbridge/impl/GBDevice.java

```diff
@@ -73,11 +74,14 @@ public GBDevice createFromParcel(Parcel source) {
     private String mModel;
     private State mState = State.NOT_CONNECTED;
     private short mBatteryLevel = BATTERY_UNKNOWN;
+    private float mBatteryVoltage = BATTERY_UNKNOWN;
     private short mBatteryThresholdPercent = BATTERY_THRESHOLD_PERCENT;
     private BatteryState mBatteryState;
     private short mRssi = RSSI_UNKNOWN;
     private String mBusyTask;
     private List<ItemWithDetails> mDeviceInfos;
+    private int mLedColor = 0;
+    private float mFmFrequency = FREQUENCY_UNKNOWN;
```

Comment:
I thought about it, and It would, yes. But that would mean they would show up on the device info list when the card is expanded. Can I add a boolean flag to make certain infos not?

Reply:
Yes, that seems appropriate. Also, this is closely related to #1176, there should be some integration between the device actions such as these and the infos.

Reply:
Yes, the device actions should be able to access and use those extra infos.